### PR TITLE
A new entry for the Lambek Calculus

### PR DESCRIPTION
**Pull request by heades**

### CODE REVIEW

File: Source/mainmatter/LambekCalc.tex [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+\end{array}
+\]
+\end{calculus}
+
+\begin{clarifications}
+The Lambek Calculus described here was introduced by Joachim Lambek to
+study sentence structure in 1958 \cite{lambek1958}.  Actually the
+calculus Lambek first introduced, despite being motivated by algebraic
+considerations as we are told in \cite{lambek1988}, had no constant
+corresponding to the unity of the tensor product $I$. The Lambek
```
Comment (Ceilican) [inline Source/mainmatter/LambekCalc.tex]: Shouldn't it be "unit" instead of "unity"?

**Comment by Ceilican** [comment]
Many thanks!

(please answer my question about a possible typo.)

File: Source/mainmatter/LambekCalc.tex (hunk 1 above)
Comment (heades) [inline Source/mainmatter/LambekCalc.tex]: Yeah, it should be "unit."  Should I fix it and do another pull request?

Comment (Ceilican) [inline Source/mainmatter/LambekCalc.tex]: Yes, please!
